exonum_launcher/supervisor.py

```diff
@@ -261,6 +263,30 @@ def _build_stop_service_change(self, instance: Instance, change: Any) -> None:
 
         change.stop_service.CopyFrom(stop_service)
 
+    def _build_resume_service_change(self, instance: Instance, change: Any) -> None:
+        """Creates a ConfigChange for resuming a service."""
+
+        assert self._service_module is not None
+        resume_service = self._service_module.ResumeService()
+
+        if instance.instance_id is None:
+            # Instance ID is currently unknown, retrieve it.
+            explorer = Explorer(self._main_client)
+            instance_id = explorer.get_instance_id(instance)
+
+            if instance_id is None:
+                raise RuntimeError(f"Instance {instance} doesn't seem to be deployed, can't change configuration")
+
+            instance.instance_id = instance_id
+
+        resume_service.artifact.runtime_id = instance.artifact.runtime_id
+        resume_service.artifact.name = instance.artifact.name
+        resume_service.artifact.version = instance.artifact.version
+
+        resume_service.instance_id = instance.instance_id
+
+        change.resume_service.CopyFrom(resume_service)
```

Comment:
I think we should encode `params` as well, similarly to [start request](https://github.com/exonum/exonum-launcher/blob/1c363494c293a3cbc24ecd94e689cbd7d19a7b37/exonum_launcher/supervisor.py#L242).